src/w_wad.c

```diff
@@ -632,48 +632,7 @@ void W_ProcessInWads(const char *name, void (*process)(int lumpnum),
                  name, process, iwad);
 }
 
-// [FG] avoid demo lump name collisions
```

Comment:
We no longer add the demo file to the wad system, but load it directly (see changes to `G_DoPlayDemo`). This way we don't have the 8.3 limitation on the demo file name. So this will work without problems:
```
woof -record map01
woof -playdemo map01
```